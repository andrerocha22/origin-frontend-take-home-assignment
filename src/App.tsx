import * as React from 'react';
import SavingGoal from './pages/SavingGoal';
import Header from './components/Header';

import './assets/styles/app.scss';

const App: React.FC = () => {
  return (
    <div className="app">
      <Header />
      <SavingGoal />
    </div>
  );
};

export default App;
