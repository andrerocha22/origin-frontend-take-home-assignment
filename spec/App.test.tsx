import React from 'react';

import App from '../src/App';

import { shallow } from 'enzyme';

import toJson from 'enzyme-to-json';

describe('<App />', () => {
  describe('render()', () => {
    test('renders the component app', () => {
      const wrapper = shallow(<App />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
