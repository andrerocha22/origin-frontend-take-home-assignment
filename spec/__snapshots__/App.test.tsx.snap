// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> render() renders the component app 1`] = `
<div
  className="app"
>
  <Header />
  <SavingGoal />
</div>
`;
